webhook-verification/js/express/index.js

```diff
@@ -1,30 +1,35 @@
 const express = require('express')
 const app = express()
 const crypto = require('crypto')
+const timingSafeCompare = require('tsscmp')
```

Comment:
How does this differ from the built in function? 

https://nodejs.org/api/crypto.html#crypto_crypto_timingsafeequal_a_b

Reply:
Yep, looked into this but had this problem:
> a and b must both be Buffers, TypedArrays, or DataViews, and they must have the same byte length.

Means we don't need to convert to buffers ourselves (which have their own security implications) and check lengths. Additionally, tsscmp uses double HMAC to further confuse this comparison - although it may not make too much of a difference in this use case given this is already a hash and the correct length would be known.

The [implementation of tsscmp](https://github.com/suryagh/tsscmp/blob/master/lib/index.js) actually uses node's crypto.timingSafeEqual if it's available.